.github/issue_template.md

```diff
@@ -0,0 +1,11 @@
+## Expected Behavior
```

Comment:
It would be amazing if we could add some instructions or ideas for what to do if it's a feature request.